### DISCUSSION TITLE
GitHub Foundations is unavailable

### DISCUSSION THREAD

**Discussion by Jacksx20** [community/community#143592] [GitHub Education] [answered]
### Hi everyone,

GitHub Foundations certification in student development package still shows charges, when can learning resume？

![图片](https://github.com/user-attachments/assets/7c4b68a3-c9d8-445a-b057-2e29358bbcf9)

**Comment by davevad93** [comment]
Hi @Jacksx20 ,

There was a bug and they had to disable the offer, technically they should have fixed it by October 31 but they haven't done it yet. Subscribe to this discussion #138834 so you can stay tuned, as soon as the free certification is available again they will announce it there.

**Reply by Jacksx20** [reply]
thanks

**Reply by Jacksx20** [reply, marked as answer]
Good news, it has been restored today
![图片](https://github.com/user-attachments/assets/af8f5842-06ce-4c2c-9e2d-4d7e541fd351)
